tests/cdrouter_bootstrap.py

```diff
@@ -58,6 +58,16 @@ def runTest(self):
 
         c = CDRouter(self.cdrouter_server)
 
+        """Go to cdrouter frequency and reboot to Dual-stack config"""
+        wan_ip = board.get_interface_ipaddr("wan0")
+        if self.config.board['cdrouter_location'] == 'CBN':
+            cm_freq = self.config.board['cdrouter_freq']
+        else:
+            cm_freq = self.config.board['common']['CM_online_freq']
```

Comment:
I thought we talked with @SyHuangCBN about this? Can't we drop this freq from the config and determine it dynamcially? (From a mac domain listed in the board config?)

Reply:
Okay, I will modified this part after mac domain listed in the board config